boards/makerbase/mks_canable_v20/mks_canable_v20.yaml

```diff
@@ -0,0 +1,20 @@
+identifier: mks_canable_v20
+name: MKS CANable V2.0
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 32
+flash: 64
+supported:
+  - can
+  - gpio
+  - rng
+  - counter
```

Comment:
Have you ran the tests for these?

Reply:
Please correct me: But CI shall at least build them.